test/e2e/common.go

```diff
@@ -46,6 +46,10 @@ func Byf(format string, a ...interface{}) {
 	By(fmt.Sprintf(format, a...))
 }
 
+func Logf(format string, a ...interface{}) {
```

Comment:
Can we use framework/internal/log everywhere?

Reply:
@cpanato Any reason do we want to expose this as an API, or do we want to keep it internal?

Reply:
@sedefsavas the on in framework is internal and only accessible in the framework and we cant access from the e2e tests package

Reply:
i can change that to be internal @vincepri

Reply:
@fabriziopandini @sedefsavas thoughts?

Reply:
Please change this to private
(also here, logging is tangential to the goal of test/e2e and we don't want to expose methods for it)

Reply:
done @fabriziopandini PTAL when you have time. thanks!